### PR TITLE
Set version for develop branch according to PEP 440

### LINKED ISSUE #2187: Set version for develop branch.
According to [PEP 440](https://www.python.org/dev/peps/pep-0440/#developmental-releases), we should set version `X.YaN.devM`.

### PR DESCRIPTION
**Pull request by workingloong**
Fix #2187

According to [PEP 440](https://www.python.org/dev/peps/pep-0440/#id24), we should set version X.YaN.devM.

### CODE REVIEW

File: elasticdl/docker/Dockerfile.ci [hunk 1]
```diff
@@ -13,22 +14,22 @@ RUN python -m pip install --quiet -r /model_zoo/requirements.txt \
         --extra-index-url=${EXTRA_PYPI_INDEX}
 
 # Install elasticdl_preprocessing package
-COPY build/elasticdl_preprocessing-develop-py3-none-any.whl /
-RUN python -m pip install --quiet /elasticdl_preprocessing-develop-py3-none-any.whl \
+COPY build/elasticdl_preprocessing-{PKG_VERSION}-py3-none-any.whl /
```
Comment (Yancey0623) [inline elasticdl/docker/Dockerfile.ci]: Maybe we can copy all the wheel files into the image, and install them in one line, multiple `RUN`, minimize the layers of Docker image can reduce the size.
``` docker
COPY build/*.whl /
RUN pip install /*.whl && rm /*.whl
```

c.f. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

File: elasticdl/docker/Dockerfile.ci [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE} as ci
 ARG EXTRA_PYPI_INDEX=https://pypi.org/simple
+PKG_VERSION=0.2.0rc3.dev1
```
Comment (Yancey0623) [inline elasticdl/docker/Dockerfile.ci]: dev0?

Comment (workingloong) [reply elasticdl/docker/Dockerfile.ci]: Yeah1

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**